CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.10.0
+
+- Replaced depreciated `Endpoint` with endpoint_url, disabled use of `Endpoint 2.0`
+- Updated `aws-config ` to 0.54.1
+- Updated `aws-sdk-athena` to 0.24.0
+- Updated `aws-sdk-s3` to 0.24.0
+- Updated `aws-sdk-sqs` to 0.24.0
+- Updated `aws-smithy-http` 0.54.4
+- Updated `aws-types` to 0.54.1
+- Updated `rust crate tokio` to 1.26.0
+- Updated `rust crate clap` to 4.1.8
+- Updated `rust crate bytesize` to 1.2.0
+- Updated `rust crate serial_test` to v1
```

Comment:
**nitpick**: There's a styling difference here between the AWS-related crates and the others, it's probably better to use a consistent style throughout:

```suggestion
- Updated rust crate `tokio` to 1.26.0
- Updated rust crate `clap` to 4.1.8
- Updated rust crate `bytesize` to 1.2.0
- Updated rust crate `serial_test` to v1
```